[2024-11-05T21:00:03.465+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-05T21:00:03.490+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: backup_files_dag.backup_files scheduled__2024-11-04T21:00:00+00:00 [queued]>
[2024-11-05T21:00:03.504+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: backup_files_dag.backup_files scheduled__2024-11-04T21:00:00+00:00 [queued]>
[2024-11-05T21:00:03.505+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-11-05T21:00:03.530+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): backup_files> on 2024-11-04 21:00:00+00:00
[2024-11-05T21:00:03.542+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=158) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-05T21:00:03.545+0000] {standard_task_runner.py:72} INFO - Started process 168 to run task
[2024-11-05T21:00:03.546+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'backup_files_dag', 'backup_files', 'scheduled__2024-11-04T21:00:00+00:00', '--job-id', '134', '--raw', '--subdir', 'DAGS_FOLDER/backup_files_dag.py', '--cfg-path', '/tmp/tmpt5l7za33']
[2024-11-05T21:00:03.548+0000] {standard_task_runner.py:105} INFO - Job 134: Subtask backup_files
[2024-11-05T21:00:03.635+0000] {task_command.py:467} INFO - Running <TaskInstance: backup_files_dag.backup_files scheduled__2024-11-04T21:00:00+00:00 [running]> on host fe1d23aa9a61
[2024-11-05T21:00:03.772+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='backup_files_dag' AIRFLOW_CTX_TASK_ID='backup_files' AIRFLOW_CTX_EXECUTION_DATE='2024-11-04T21:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-04T21:00:00+00:00'
[2024-11-05T21:00:03.775+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-05T21:00:03.776+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-05T21:00:03.777+0000] {logging_mixin.py:190} INFO - Current task name:backup_files state:running start_date:2024-11-05 21:00:03.491677+00:00
[2024-11-05T21:00:03.779+0000] {logging_mixin.py:190} INFO - Dag name:backup_files_dag and current dag run status:running
[2024-11-05T21:00:03.780+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-05T21:00:03.782+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-11-05T21:00:03.784+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cp -r /opt/***/source* /opt/***/backup']
[2024-11-05T21:00:03.799+0000] {subprocess.py:86} INFO - Output:
[2024-11-05T21:00:08.352+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-11-05T21:00:08.402+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-05T21:00:08.403+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=backup_files_dag, task_id=backup_files, run_id=scheduled__2024-11-04T21:00:00+00:00, execution_date=20241104T210000, start_date=20241105T210003, end_date=20241105T210008
[2024-11-05T21:00:08.440+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-05T21:00:08.441+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-05T21:00:08.442+0000] {logging_mixin.py:190} INFO - Dag name:backup_files_dag queued_at:2024-11-05 21:00:00.993556+00:00
[2024-11-05T21:00:08.444+0000] {logging_mixin.py:190} INFO - Task hostname:fe1d23aa9a61 operator:BashOperator
[2024-11-05T21:00:08.468+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-05T21:00:08.491+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-05T21:00:08.496+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
